ics/tool_install/linux_install_ics_tools_arch.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Details
+echo 'Install Common ICS Tools on Linux Systems'
+echo '   linux_install_ics-tools.sh brought to you by Cutaway Security, LLC.'
+echo '   Author: Don C. Weber (@cutaway)'
+echo 'WARNING: Automated install of third-party software and tools we do not control.'
+echo 'WARNING: No warranty or guarantee these tools are secure or do not contain malicious code.'
+echo 'WARNING: Check all installed software on your own before use.'
+echo 'WARNING: USE AT YOUR OWN RISK.'
+echo
+read -n1 -s -r -p $'Press Y to continue...\n' key
+
+if [ $(tr '[:lower:]' '[:upper:]' <<< "$key") = 'Y' ]; then
+    echo 'Excellent.... Continuing... Enjoy....'
+else
+    echo 'Exiting....'
+    exit 0
+fi
+
+# Change to User's Home directory and create Tools Dir
+echo 'Generating ICS Tools Directory at ~/Tools/ics-tools'
+cd ~
+TOOLDIR=$HOME'/Tools/ics-tools'
+if [ ! -d "${TOOLDIR}" ]; then
+    mkdir -p $TOOLDIR
+ else
+    echo "ICS Tools Directory found. Exiting...."
+    exit 0
+fi
+
+# Update 
+echo 'Updating System Packages'
+sudo pacman -Syu
```

Comment:
Probably need to think about how to handle different OSes and Distro. Would rather move to a configuration file first and build in logic about different OSes.